stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -358,7 +357,7 @@ internal constructor(
         val cacheableTokenProvider = CacheableTokenProvider(tokenProvider)
         val userState = userStateService.state
 
-        ClientState.get().toMutableState()?.setUser(user)
+        clientState.toMutableState()?.setUser(user)
```

Comment:
At this point we are not sure the user is set, it can fails

Reply:
This logic is not changed, it is only accessing `clientState` in a different way. This line is not requesting the user, it actually setting it.

Reply:
This code changed already.